src/services/explore.ts

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import { UUID } from 'crypto';
+import { backendFetch } from './utils';
+
+const PREFIX_EXPLORE_SERVER_QUERIES =
+    import.meta.env.VITE_API_GATEWAY + '/explore';
+
+export function createFilter(
```

Comment:
Why don't you define these methods as hooks ? So you can import the token directly here and you make it transparent for the components that use APIs ?

Reply:
Personnaly I prefer like that, otherwise that means we encapsulate every fetch in a hook, it's a bit of boilerplate code for me.
We can discuss about to take a decision if you want.

Reply:
I agree it's a bit of additionnal code, but I don't like that you need to retrieve userToken in functional components. Functional components should remain functional and when you call createFilter you want to pass only functional arguments to this method, the management of the token should be handled somewhere else in a purely technical layer.

Reply:
Ok let's discuss about it

Reply:
Decision has been made to use a global var to store the redux store and have access to the token from the API files directly.
Temporary solution before refactoring the whole token management.